content/posts/euc/hub-templates.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Hub Service Template"
+date: 2020-10-24T00:00:00-00:00
+draft: false
+categories: [HUBSERVICES, EUC]
+author: Nagul Subramanian
+authorlink: /https://www.linkedin.com/in/nagulsubramanian 
+tags:
+- Access
+- Hub Services
+- Templates 
+- Workspace ONE
+lightgallery: true
+---
+
+## What is Hub Services? 
+To simply state it helps enterprise introduce a true digital experience to their employees on any device or web. This is just getting started with lot of potential to unlock features to make their employee experience a world class. 
+
+There are few docs that captures all the features in Hub Services in detail as referenced below, so with that I want to focus specifically on the Hub Services template in this post. 
+- [Enabling and Delivering Hub Services with Workspace ONE Intelligent Hub](https://kb.vmware.com/s/article/2960170)
+- [VMware Workspace ONE Intelligent Hub Series Part 1: Evolve from Legacy App Catalog as Web Clip to Hub Catalog](https://thomascheng.net/2020/07/17/evolve-from-legacy-app-catalog-as-web-clip-to-hub-catalog-within-vmware-workspace-one-intelligent-hub/)
+## Why we need the Hub Services Templates?
+
+- To truly provide an user-centric experience we need a way to personalize/tailor content for each personas. Let's take the knowledge worker and the sales personas as an example, each of their day-to-day functions differ and rely on unique resources to accomplish their task. Such that it is important to provide a curated experience and that can achieved via the Hub Services template. As the name represents, it allows you to create templates with different set of features and/or configurations and can then be deployed to UEM smart groups or Access user groups or both 
```

Comment:
Would it make sense to touch briefly on why Templates were built? Hub services were a global setting and could not provide customization per group or a subset of devices.